src/modules/reservations.js

```diff
@@ -0,0 +1,91 @@
+import {
+  nameofperson, commentList,
+} from './comments.js';
+import updateCommentCount from './updateCommentCount.js';
+
+const popupContainer = document.querySelector('#pop-up-container');
+const FORM = document.querySelector('form');
+
+export const validation = (form) => {
+  const numElements = form.elements.length;
+  if (numElements === 4) {
+    const newInput1 = form.querySelector('#date1');
+    const newInput2 = form.querySelector('#date2');
+    if (newInput1.style.border === '2px solid red' || newInput2.style.border === '2px solid red') {
+      return false;
+    }
+    return true;
+  }
+  return true;
+};
+
+export const ResetReservations = (form) => {
+  // const form = form.querySelector('form');
+  form.reset();
+
+  // inputs.forEach((input) => {
+  //   input.value = '';
+  //   input.style.border === 'none';
+  // });
+  // return true;
+};
```

Comment:
Great job team 🤝

- [ ] [Optional] I would also recommened that you remove commented code from your project as this is regarded as bad practice.